src/components/ui/Link/link.scss

```diff
@@ -0,0 +1,40 @@
+@import "../../../styles/vendors/include-media.scss";
+
+[data-store-link] {
+  min-width: auto;
+  min-height: auto;
+  padding: var(--space-2) var(--space-0);
+  text-decoration: none;
+  &:hover { text-decoration: underline; }
+  &:visited { color: var(--color-link-visited); }
+}
+
+// Links ////////////////////////////
+
+.link {
+  color: var(--color-link);
+  line-height: 1.5;
+}
+
+.link-display {
+  color: var(--color-secondary-4);
+  line-height: 1.5;
+}
+
+.link-inline {
```

Comment:
hmm, here I would suggest or follow BEM pattern to name these classes or create `data-atribule` with variant to style it. What do you think?